frontend/server/src/Controllers/Problem.php

```diff
@@ -232,6 +232,28 @@ private static function validateCreateOrUpdate(
                     'visibility'
                 );
             }
+            if (
+                ($problem->visibility === \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING ||
+                  $problem->visibility === \OmegaUp\ProblemParams::VISIBILITY_PRIVATE_WARNING) &&
+                    !is_null($params->visibility) &&
+                    $problem->visibility !== $params->visibility &&
+                    !\OmegaUp\Authorization::isQualityReviewer($identity)
+            ) {
+                if (
+                    intval(
+                        $params->visibility
+                    ) !== \OmegaUp\ProblemParams::VISIBILITY_PRIVATE_WARNING &&
+                    intval(
+                        $params->visibility
+                    ) !== \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING &&
+                    \OmegaUp\Authorization::isProblemAdmin($identity, $problem)
```

Comment:
antes en este método ya se validó `\OmegaUp\Authorization::canEditProblem()` que es ya sea que es un revisor de calidad o es un administrador de problema. en L240 se determinó que _no_ es un revisor, así que eso sólo deja ser administrador de problema.

entonces esta validación es redundante y se puede eliminar.

```suggestion
                    ) !== \OmegaUp\ProblemParams::VISIBILITY_PUBLIC_WARNING
```